internal/impl/provisioner/endpoint_provisioner_impl.go

```diff
@@ -295,10 +295,8 @@ func (provisioner *endpointProvisionerImpl) Deprovision(queueName string, ignore
 		return solace.NewError(&solace.IllegalStateError{}, constants.UnableToDeprovisionParentServiceNotStarted, nil)
 	}
 
-	properties := provisioner.properties.GetConfiguration() // we don't need all these properties for deprov
-
-	// Override defaults durability to be True since we currently only support durable
-	// properties[config.EndpointPropertyDurable] = true
+	// we don't need all these properties for deprov
+	properties := provisioner.properties.GetConfiguration()
```

Comment:
@TrentDaniel, I removed this commented out code since it is not useful (it was forgotten from the endpoint provisioner Epic when we decided to delegate setting the endpoint durability to ccsmp) and we shouldn't be having commented out code checked into source control.

Reply:
Sounds good